src/status_im2/contexts/quo_preview/wallet/account_overview.cljs

```diff
@@ -0,0 +1,92 @@
+(ns status-im2.contexts.quo-preview.wallet.account-overview
+  (:require [quo2.components.wallet.account-overview.view :as quo2]
```

Comment:
import from `quo2.core` we want to treat `quo2` components as it's own module. i,e any uses of it outside of that scope should come from `quo2.core` 👍